src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -35,6 +35,8 @@
     int shipsHit = 2;
     int shoots=0;
     int hitSearch = 0, direction = 1, inc = 0;
+    int hardAI = 0;
```

Comment:
I like this, it's a good way to keep track of whether we're in hard or easy. Maybe change variable name to "difficulty", as we're going to use it to switch between easy and hard.